apps/src/templates/SchoolNotFound.jsx

```diff
@@ -0,0 +1,201 @@
+import React, { Component, PropTypes } from 'react';
+import color from "../util/color";
+import i18n from "@cdo/locale";
+import { STATES } from '../geographyConstants';
+
+const styles = {
+  question: {
+    fontSize: 16,
+    fontFamily: '"Gotham 3r", sans-serif',
+    color: color.charcoal,
+    paddingTop: 10,
+    paddingBottom: 5
+  },
+  input: {
+    height: 40,
+    width: 250,
+    fontFamily: '"Gotham 3r", sans-serif',
+    padding: 5
+  },
+  dropdown: {
+    fontFamily: '"Gotham 4r", sans-serif',
+    color: color.charcoal,
+    height: 30,
+    marginTop: 5,
+    width: 250
+  },
+  errors: {
+    fontSize: 14,
+    fontFamily: '"Gotham 3r", sans-serif',
+    color: color.red,
+    paddingTop: 5,
+    paddingBottom: 5
+  },
+  field: {
+    float: 'left',
+    height: '80px',
+    width: '450px',
+  },
+  clear: {
+    width: '100%',
+    clear: 'both'
+  },
+  asterisk: {
+    fontSize: 20,
+    fontFamily: '"Gotham 5r", sans-serif',
+    color: color.red,
+  },
+};
+
+export default class SchoolNotFound extends Component {
+  static propTypes = {
+    setField: PropTypes.func,
+    schoolName: PropTypes.string,
+    schoolCity: PropTypes.string,
+    schoolState: PropTypes.string,
+    schoolZip: PropTypes.string,
+    schoolType: PropTypes.string,
+    showErrorMsg: PropTypes.bool
+  };
+
+  sendNameToParent = (event) => {
+    this.props.setField("schoolName", event.target.value);
+  }
+
+  sendTypeToParent = (event) => {
+    this.props.setField("schoolType", event.target.value);
+  }
+
+  sendStateToParent = (event) => {
+    this.props.setField("schoolState", event.target.value);
+  }
+
+  sendZipToParent = (event) => {
+    this.props.setField("schoolZip", event.target.value);
+  }
+
+  sendCityToParent = (event) => {
+    this.props.setField("schoolCity", event.target.value);
+  }
+
+  render() {
+    const schoolTypes = [
+      '',
+      i18n.schoolTypeCharter(),
+      i18n.schoolTypePrivate(),
+      i18n.schoolTypePublic(),
+      i18n.other()
+    ];
+
+    return (
```

Comment:
This is a long render method.  This is fine to do later, but it'd be good to find duplication within this block that can be eliminated with pure functional components, even within the same file.  For example, I think we might be able to break out a `Field` component?

```jsx
const Field = ({name, required, children}) => (
  <div style={styles.field}>
    <label>
      <div style={styles.question}>
        {name}
        {required && <span style={styles.asterisk}> *</span>}
      </div>
      {children}
    </label>
  </div>
);

// Then in the main render method:
<div>
  <Field name={i18n.schoolName()} required>
    <input
      id="school_name"
      type="text"
      name="school_name_s"
      value={this.props.schoolName}
      onChange={this.sendNameToParent}
      style={styles.input}
    />
  </Field>
  <Field name={i18n.schoolType()} required>
    <select
      name="school_type_s"
      value={this.props.schoolType}
      onChange={this.sendTypeToParent}
      style={styles.dropdown}
    >
      {schoolTypes.map((schoolType, index) =>
        <option
         value={schoolType}
         key={index}
       >
        {schoolType}
       </option>
     )}
    </select>
  </Field>
</div>
```